embedded-azurite/src/main/java/com/playtika/test/azurite/AzuriteProperties.java

```diff
@@ -0,0 +1,22 @@
+package com.playtika.test.azurite;
+
+import com.playtika.test.common.properties.CommonContainerProperties;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+@Data
+@EqualsAndHashCode(callSuper = true)
+@ConfigurationProperties("embedded.azurite")
+public class AzuriteProperties extends CommonContainerProperties {
+
+    static final String AZURITE_BEAN_NAME = "azurite";
+
+    /**
+     * Using the latest version on purpose here, because there also always exists always one version of Azure.
+     * If you want a fixed version, check https://hub.docker.com/_/microsoft-azure-storage-azurite
+     */
+    String dockerImageVersion = "latest";
```

Comment:
Please replace with the fixed version. 
Otherwise, it may fail when the newest version of container is not compatible with the setup in testcontainers-spring-boot project.